app/views/providers/means_summaries/_income_cash_payments.html.erb

```diff
@@ -0,0 +1,54 @@
+<!-- TODO: share with means report?! -->
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h3 class="govuk-heading-m"><%= t('.heading') %></h3>
+    </div>
+    <div class="govuk-grid-column-one-third govuk-summary-list--no-border align-text-right">
+        <p>
+            <%= link_to_accessible(t('generic.change'),
+                                   check_answer_url_for(:providers,
+                                                        :cash_incomes,
+                                                        @legal_aid_application),
+                                   class: 'govuk-link change-link', suffix: t('.heading')) %>
+        </p>
+    </div>
+</div>
+
+<%
+credits = @legal_aid_application.legal_aid_application_transaction_types.credits
+cash_credits = @legal_aid_application.cash_transactions.credits.order(:transaction_date)
+credit_categories = TransactionType.credits.not_children
+%>
+
+<% if !@legal_aid_application.transaction_types.credits.exists? %>
```

Comment:
How do you feel about switching this round? ([docs](https://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-none-3F))

```suggestion
<% if @legal_aid_application.transaction_types.credits.none? %>
```